site/src/pages/WorkspaceSettingsPage/WorkspaceSchedulePage/WorkspaceSchedulePage.stories.tsx

```diff
@@ -11,15 +11,15 @@ import { templateByNameKey } from "api/queries/templates";
 import { workspaceByOwnerAndNameKey } from "api/queries/workspaces";
 import type { Workspace } from "api/typesGenerated";
 import {
-	reactRouterNestedAncestors,
+	reactRouterOutlet,
 	reactRouterParameters,
 } from "storybook-addon-remix-react-router";
 import { WorkspaceSettingsLayout } from "../WorkspaceSettingsLayout";
 import WorkspaceSchedulePage from "./WorkspaceSchedulePage";
 
 const meta = {
 	title: "pages/WorkspaceSchedulePage",
-	component: WorkspaceSchedulePage,
+	component: WorkspaceSettingsLayout,
```

Comment:
I'd prefer that this `component` stay "correct", but the way the react-router storybook addon works really confuses me, so I don't really have a better idea.